src/protocol/kw2.rs

```diff
@@ -16,9 +16,6 @@ impl Kw2 {
     let start = Instant::now();
 
     loop {
-      o.write_all(&[0x04])?;
-      o.flush()?;
```

Comment:
Are you sure this isn't needed? As far as I can remember, it *does* work without this, but the response time is lower if you send `0x04` before waiting for `0x05`.

Reply:
From my understandings of the [documentaion](https://github.com/openv/openv/wiki/Protokoll-300)  the '0x04' resets the Optolink connection. The trick is to the reset the Optolink before every command to get a faster '0x05' (SYNC), right? Ok, that makes sense. I will undo it. But should then the 'negotiate' function send another '0x04', or not?

Reply:
> The trick is to the reset the Optolink before every command to get a faster '0x05' (SYNC), right?

Yes, exactly.

> But should then the 'negotiate' function send another '0x04', or not?

I guess it doesn't hurt to send `0x04` when initializing, so I would suggest keeping the `0x04` in `negotiate` and then re-using `negotiate` inside `sync`.

Reply:
And maybe add a comment explaining that this is in order to get a faster `SYNC´ response.